### DISCUSSION TITLE
Ideas/Suggestions

### DISCUSSION THREAD

**Discussion by blackdagor28** [pulse-browser/browser#142] [Ideas]
Hi! My name is Diego, I was looking for other forks of Firefox that can have what EXACTLY Firefox needs to keep up their game, and I found your project, it's awesome to know there's still people who really knows what Firefox needs, and I really want Pulse to success in a future.

New features is what mostly users are asking for, and the devs are not responding well to this, the two main voted ideas are getting postponed, I don't know why, but maybe (only if it's possible) you could start with those ideas first, that is my main suggestion, I'm sure that you will catch a lot of users by doing that.

Alongside my first suggestion, there's something that could be a good idea too, I submitted it in Mozilla Ideas and I got some Kudos, but maybe you could find this a good idea too (of course as a user interested in this project, I completely respect what you decide), I think the TTS that Firefox uses it's old and maybe deprecated, Reader Mode needs a new face, similar to Edge would be awesome, with some new voices (not robotic), I'm sure Audiobook users will be thankful for that feature haha.

Thank you so much for your time reading this, and completely appreciate your project, I'll get into Discord.
Sincerely, Diego.

**Comment by niutech** [comment]
Please enable by default the experimental support for:
- [Compact mode](https://support.mozilla.org/en-US/kb/compact-mode-workaround-firefox)
- JPEG XL images
- inert HTML attribute
- password display
- initial-letter CSS property
- backdrop-filter CSS property
- fit-content() CSS function
- container query length units
- scroll-linked animations
- :has() CSS pseudo-class
- OffscreenCanvas
- WebGPU
- selectAudioOutput() function
- asynchronous SourceBuffer
- OpenFont COLRv1
- ClipboardItem and ClipboardRead
- HTML Sanitizer API
- Payment Request API
- Prioritized Task Scheduling API
- Permissions-Policy header
- Clear-Site-Data: "cache" header
- Web console context selector
